content/en/getting_started/synthetics/private_location.md

```diff
@@ -0,0 +1,60 @@
+## Set up your first private location
```

Comment:
Can we have a bigger font here? So that it looks like an actual page title? Like we have in https://docs.datadoghq.com/getting_started/application/monitors for instance?

Reply:
And can we add more styling to this section (like some bold parts?)? To make it look as pleasant as possible to read